xbmc/Application.h

```diff
@@ -252,38 +260,42 @@ class CApplication : public CXBApplicationEx, public IPlayerCallback, public IMs
 
   /*!
    \brief Starts a video library cleanup.
-   \param userInitiated Whether the action was initiated by the user (either via GUI or any other method) or not.  It is meant to hide or show dialogs.
-   \param content Content type to clean, blank for everything
+   \param userInitiated Whether the action was initiated by the user (either via GUI or any other
+   method) or not.  It is meant to hide or show dialogs. \param content Content type to clean, blank
+   for everything
    */
   void StartVideoCleanup(bool userInitiated = true, const std::string& content = "");
 
   /*!
    \brief Starts a video library update.
    \param path The path to scan or "" (empty string) for a global scan.
-   \param userInitiated Whether the action was initiated by the user (either via GUI or any other method) or not.  It is meant to hide or show dialogs.
-   \param scanAll Whether to scan everything not already scanned (regardless of whether the user normally doesn't want a folder scanned).
+   \param userInitiated Whether the action was initiated by the user (either via GUI or any other
+   method) or not.  It is meant to hide or show dialogs. \param scanAll Whether to scan everything
+   not already scanned (regardless of whether the user normally doesn't want a folder scanned).
```

Comment:
The wrapping of all `\param` into one block makes it less readable, can this be avoided?

Reply:
Good catch. I had missed looking at the comments. Updated the `.clang-format` file to not reflow comments and it looks like it solves the issue.